toolkit/tools/imagecustomizerapi/overlay.go

```diff
@@ -5,26 +5,38 @@ package imagecustomizerapi
 
 import (
 	"fmt"
+	"path"
 	"strings"
 )
 
 type Overlay struct {
-	LowerDir  string               `yaml:"lowerDir"`
-	UpperDir  string               `yaml:"upperDir"`
-	WorkDir   string               `yaml:"workDir"`
-	Partition *IdentifiedPartition `yaml:"partition"`
+	LowerDir        string  `yaml:"lowerDir"`
+	UpperDir        string  `yaml:"upperDir"`
+	WorkDir         string  `yaml:"workDir"`
+	MountPoint      string  `yaml:"mountPoint"`
+	IsRootfsOverlay bool    `yaml:"isRootfsOverlay"`
+	MountDependency *string `yaml:"mountDependency"`
```

Comment:
Does `MountDependency` and `MountOptions` need to be `*string`? Can they be just `string` instead?